cantools/database/can/formats/arxml/system_loader.py

```diff
@@ -1440,8 +1439,7 @@ def _load_signal(self, i_signal_to_i_pdu_mapping):
             return None
 
         # Default values.
-        initial = None
-        invalid = None
+        raw_initial = None
         minimum = None
         maximum = None
```

Comment:
shouldn't `.minimum` and `.maximum` be handled the same way as `.invalid` and `.initial`?